core/scripts/vrfv2/testnet/main.go

```diff
@@ -56,6 +63,121 @@ func main() {
 	e := helpers.SetupEnv(false)
 
 	switch os.Args[1] {
+	// "generate-proof" generates a proof for a given request ID and block number
+	// using they VRF key from the given VRF node. It can optionally submit the
+	// transaction on-chain. This scripts is useful when you want to manually fulfill
+	// a request ID but do not already have a proof
+	// this script exports key from the given node and decrypts it in memory
+	// the exported key is not persisted locally
```

Comment:
Will this script only be executed somewhere in a safe environment, e.g. on the CLL infrastructure with direct access to the CL node? Just a thought of precaution, if you can execute this from your laptop, then the private key for proofs might be exposed to in-memory attacks (AVTs or advanced volatile threats). BTW what is the recovery scenario in case the private key for generating proofs ever gets hijacked from the CL node, can we replace it with another one?